### ISSUE TITLE
Integrate library into sparsebundlefs instance

### ISSUE THREAD

**Issue by noperator** [jief666/sparsebundlefs-lib#1]
@jief666 , I just compiled your fork of `sparsebundlefs` (thanks for the `openssl-1.0` tip, @KenMacD) and have been having issues getting the subsequent `mount` command to work properly. Assuming you'd still be working on a solution for mounting encrypted sparsebundles, I followed your commit history and found this repo. Have you actually integrated `sparsebundlefs-lib` into an instance of `sparsebundlefs`? Or simply provided a library for someone else to do so? Thanks for trying to solve this narrow issue—great work!

**Comment by jief666** [comment]
yes, sparsebundlefs-lib is a "subset" of sparsebundlefs to allow integration in other project.
To be able to mount, the simpler is to use sparsebundle with no partition map. Otherwise you have to use the losetup thing that I have never figured out.
I moved to another project that mount anything HFS, APFS, sparsebundle, encrypted or not, any partition map, etc. Also in a simpler way because only one command is needed. Have a look to https://github.com/jief666/hdimount/tree/wip.

**Comment by noperator** [comment]
I finally figured out how to mount the decrypted sparse bundle disk image using `losetup`. Here's what worked for me, for your reference: https://gist.github.com/noperator/6f2633dc952d5ae851eda3c12a608aeb

**Comment by stefan2904** [comment]
Since it is now offline, here is a "mirror" of the mentioned gist:
https://github.com/torarnv/sparsebundlefs/issues/9#issuecomment-578554651

**Comment by noperator** [comment]
Fixed the link to the gist—should work now.
